app/helpers/users_helper.rb

```diff
@@ -0,0 +1,8 @@
+module UsersHelper
+  def gravatar_for user
+    gravatar_id = Digest::MD5::hexdigest(user.email.downcase)
```

Comment:
```suggestion
    gravatar_id = Digest::MD5::hexdigest user.email.downcase
```
please add end line (blank line) for this file